lib/src/require.dart

```diff
@@ -0,0 +1,16 @@
+import 'bool_extensions.dart';
+
+/// Util for checking
+/// `since 0.7.0`
+class Require {
+  Require._();
+
+  /// Throws an exception with [message] if [check] is [false]
+  /// `since 0.7.0`
+  static bool require(bool check, String message) {
+    if (check.isFalse) {
+      throw Exception(message);
+    }
+    return check;
```

Comment:
Is there value in returning the check?

Reply:
Just to keep it consistent with `RequireFile`. In the future I see `require` allowing `object` and functions, in those cases it's useful to return the `check`. A good example would be a `Require.nonNull` that takes and object and returns it if it's not null.

Reply:
yeah it's not useful for `require` method. Will remove